packages/statistics/migrations/sqls/20210219123130-statistics-geo-country-intial-up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS "statisticsGeoCountry" (
+  "code" citext PRIMARY KEY NOT NULL,
+  "name" citext NOT NULL,
+  "searchNames" json,
```

Comment:
```suggestion
  "searchNames" jsonb,
```

[As I understand `json` is parsed whenever queried while `jsonb` is stored in parsed manner and won't require "reparsing" when queried](https://www.citusdata.com/blog/2016/07/14/choosing-nosql-hstore-json-jsonb/).